GliaWidgets/View/Chat/Message/Content/File/Download/ChatFileDownloadContentStyle.swift

```diff
@@ -1,16 +1,47 @@
 import UIKit
 
+/// Style of a  download view.
```

Comment:
Two spaces, and should say that this style is used when dealing with file attachments.